skill_markII_audio_receiver/__init__.py

```diff
@@ -49,6 +49,26 @@ def __init__(self, *args, **kwargs):
         self.renaming_airplay = False
         self.renaming_spotify = False
 
+    def initialize(self):
+        airplay_name = self.airplay_name
+        self.log.info("Initializing Mark II Audio Receiver Skill with settings:")
+        self.log.info(
+            f"Renaming Raspotify device to settings value of {self.raspotify_name}"
+        )
+        self.bus.emit(
+            Message(
+                "neon.phal.plugin.audio.receiver.set.raspotify.name",
+                {"name": self.raspotify_name},
```

Comment:
adding context `{"skill_id": self.skill_id}` should resolve the test failures here..